datadog_lambda/wrapper.py

```diff
@@ -81,13 +86,21 @@ def __init__(self, func):
             self.logs_injection = (
                 os.environ.get("DD_LOGS_INJECTION", "true").lower() == "true"
             )
+            self.merge_xray_traces = (
+                os.environ.get("DD_MERGE_XRAY_TRACES", "false").lower() == "true"
+            )
+            self.handler_name = os.environ.get("_HANDLER", "handler")
+            self.function_name = os.environ.get("AWS_LAMBDA_FUNCTION_NAME", "function")
+            self.propagator = HTTPPropagator()
 
             # Inject trace correlation ids to logs
             if self.logs_injection:
                 inject_correlation_ids()
 
-            # Patch HTTP clients to propagate Datadog trace context
-            patch_all()
+            if not dd_tracing_enabled:
+                # When using dd_trace_py it will patch all the http clients for us,
```

Comment:
If using ddtrace, does user still need to explicitly call `patch_all` in their function code? If yes, is there any downside for us to call the ddtrace patch_all for here?

Reply:
I've added the ddtrace.patch_all call here